client/src/reducers/index.js

```diff
@@ -1,4 +1,14 @@
 import { combineReducers } from 'redux';
+import { connectRouter } from 'connected-react-router';
+
 import auth from './auth';
+import alert from './alert';
+
+const createRootReducer = history =>
+  combineReducers({
+    router: connectRouter(history),
+    auth,
+    alert,
+  });
```

Comment:
🔮  Does this mean we're gonna end up having "one reducer to rule them all"?

Reply:
It's the so-called "root reducer" which is just the root of all of the react state.